tests/clients/test_aws_cost_usage_client.py

```diff
@@ -0,0 +1,34 @@
+from datetime import date
+from tests.aws_scanner_test_case import AwsScannerTestCase
+from unittest.mock import Mock
+
+from src.clients.aws_cost_usage_client import AwsCostUsageClient, CostUsageException
+from botocore.exceptions import BotoCoreError
+from tests.clients.test_aws_cost_usage_responses import GET_USAGE_COST_SUCCESS
+
+
+class TestAwsCostUsageClient(AwsScannerTestCase):
+    def test_get_aws_cost_usage_success(self) -> None:
+        boto_cost_usage = Mock(get_cost_and_usage=Mock(return_value=GET_USAGE_COST_SUCCESS))
+        client = AwsCostUsageClient(boto_cost_usage)
+
+        today = date.today()
+        expected = {
+            "Service": "Lambda",
+            "DateRange": {
+                "Start": "2021-02-01",
+                "End": f"{today.year}-{'%02d' % today.month}-{'%02d' % today.day}",
```

Comment:
This is mirroring the production code in the test code. Also relying on dynamic dates makes tests brittle. It would be better to use the predictable mocked today's date that's in `AwsScannerTestCase` and hardcode the `expected` dict accordingly. I think you just need to remove the `from datetime import date` import from `test_aws_cost_usage_client.py` and the mocked today's date will automatically be used.